app/services/storage.js

```diff
@@ -39,6 +39,14 @@ interface ProfileSchema {
   createOrUpdate(storage: ProfileStorage): Promise<number>;
 }
 
+export const SessionStartReason = {
```

Comment:
Maybe that looks like a `typedef SessionStartReason = number` and using the SSR typedef in the function declaration?